### ISSUE TITLE
PostgreSQL Support

### ISSUE THREAD

**Issue by miguelocarvajal** [Devskiller/hbm2ddl-maven-plugin#13]
Tried using this with PostgreSQL but I get the following error:

```
[ERROR] Failed to execute goal com.devskiller.jpa2ddl:jpa2ddl-maven-plugin:0.9.5:generate (default-cli) on project cci-data: Unable to build DatabaseInformation: Table "PG_CLASS" not found; SQL statement:
[ERROR] select relname from pg_class where relkind='S' [42102-196]
```

Using the org.hibernate.dialect.PostgreSQLDialect, if I switch it to org.hibernate.dialect.MySQL57Dialect (as in the example), I do not get this error.

Is there a workaround for this issue?

Thanks!

**Comment by shexiaogui** [comment]
got the same error when using postgresql

**Comment by jkubrynski** [comment]
It works OK with `CREATE` mode, the problem is only in `UPDATE`. We're trying to overcome lack of `PG_TABLE`

**Comment by jkubrynski** [comment]
Released in 0.9.9
